libcnb-data/src/buildpack.rs

```diff
@@ -260,16 +266,16 @@ libcnb_newtype!(
     /// ```
     StackId,
     StackIdError,
-    r"^([[:alnum:]./-]+|\*)$"
+    r"^[[:alnum:]./-]+$"
 );
 
 #[derive(thiserror::Error, Debug)]
 pub enum BuildpackTomlError {
     #[error("Found `{0}` but value MUST be in the form `<major>.<minor>` or `<major>` and only contain numbers.")]
     InvalidBuildpackApi(String),
 
-    #[error("Stack with id `*` MUST not contain mixins. mixins: [{0}]")]
-    InvalidStarStack(String),
+    #[error("Stack with id `*` MUST NOT contain mixins, however the following mixins were specified: {0}")]
+    InvalidAnyStack(String),
```

Comment:
Ideally, this would not be a `String` but a `Vec<String>` (or `Vec<Mixin>` when we come to it). I don't know from the top of my head if `thiserror` can deal with this out of the box or if would have to provide a `Display` implementation for it.

This isn't critical, but worth looking into if it's easily fixable.

Reply:
It turns out thiserror does allow doing this :-)
ba185605446ffb82d923b6e136e3540e2bb6335d

Reply:
Awesome! Thanks for looking into it!